subcommands/upgrade

```diff
@@ -40,6 +40,16 @@ service-upgrade-cmd() {
 
   service_commit_config "$SERVICE"
```

Comment:
Maybe it would make sense to pull before committing the config. What do you think?